.github/workflows/CI.yml

```
@@ -13,13 +18,11 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.0'
-          - '1'
-          - 'nightly'
+          - "1"
+          - "1.7"
```

```suggestion
          - "1.6"
```

Shouldn't we test on the LTS version?